code/index.html

```diff
@@ -1,12 +1,116 @@
 <!DOCTYPE html>
-<html>
-  <head>
-    <meta charset="utf-8" />
-    <title>Project Name</title>
-    <!-- dont forget the link to the css file -->
-    <!-- dont forget the meta tag for responsiveness -->
-  </head>
-  <body>
-    <!-- Build your site up as you want it! Make sure to use the form element and try to send that form to httpbin by adding action="https://httpbin.org/anything" and method="POST" to your form tag. -->
-  </body>
-</html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Forms project</title>
+    <link rel="stylesheet" type"text/css" href="style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Josefin+Sans:wght@300;400;500&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Shadows+Into+Light&display=swap" rel="stylesheet">
+    <link rel='icon' href='Logo1.png' type='image/x-icon'/ >
+</head>
+
+
+<body>
+  <header>
+    <div class="hamburger">
+        <div></div>
+        <div></div>
+        <div></div>
+    </div>
+    <img src="LOGO-RVB.png" alt="Logo Coding Club">
+    <nav>
+        <a href="./index.html">home</a>
+        <a href="https://www.bfh.ch/codingclub">mehr info</a>
+        <a href="https://www.epfl.ch/education/education-and-science-outreach/fr/promotion-education-sciences/promotion-des-sciences/coding-club-for-girls/">noch mehr info</a>
+    </nav>
+    <a href="https://www.instagram.com/codingclubforgirls/"><img src="Icon_Instagram.png"</img></a>
+  </header>
+
+  <form action="http://httpbin.org/anything" method="POST">
+    <h1>&#10024 Anmeldung "Coding Club for Girls" 👩🏽‍💻</h1>
+
+    <fieldset>
+      <legend>Allgemeine Info</legend>
+      <label>
+        <span>Name</span>
+        <input type="text" name="name" placeholder="Dein Vor- und Nachname" required>
```

Comment:
When you have several attributes, it's nice to give them new lines to make it more readable:
```
<input
  type="text"
  name="name"
  placeholder="Dein Vor- und Nachname"
  required>

```